api/comments.rb

```diff
@@ -62,5 +62,4 @@
       pass
     end
   end
-  comment.to_hash.to_json
```

Comment:
@robrap Okay, I found out a little bit more about what's going on here. [When a comment is deleted](https://github.com/edx/edx-platform/blob/master/lms/lib/comment_client/models.py#L157) it is also [updated from the response](https://github.com/edx/edx-platform/blob/master/lms/lib/comment_client/models.py#L101) that is returned by the comments service. I am not 100% sure why a deleted comment also needed to be accurate, but it seems to be a deliberate decision here.